### PR TITLE
[DEV-11525] patch requests

### PR DESCRIPTION
**Pull request by nateshim-indico**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM python:3.9.5
+FROM python:3.10.4
 
-ENV INDICO_HOST="dev.indico.io"
+ENV INDICO_HOST="dev-ci.us-east-2.indico-dev.indico.io"
```
Comment (jacobmanderson) [inline Dockerfile:3]: do we need to update this?

**Review by jacobmanderson — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (goatrocks) [reply Dockerfile:3]: good to do for tests I suppose

Comment (nateshim-indico) [reply Dockerfile:3]: yeah I figured it would be good if integration tests are run in deploy step
